CHANGELOG.rst

```diff
@@ -1,5 +1,9 @@
 Changelog for Pulse3D
 =====================
+0.25.1 (2022-08-24)
```

Comment:
Don't need to release a new version / update the changelog if just updating the docs. But keep in mind that when you do need to release a new version, the version in setup.py must also be bumped